framework/src/wifi_manager/wifi_manager_dhcp_internal.h

```diff
@@ -0,0 +1,75 @@
+/****************************************************************************
+ *
+ * Copyright 2019 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+
+#include <tinyara/config.h>
+#include <stdlib.h>
+#include <string.h>
+#include <pthread.h>
+#include <sys/types.h>
+#include <sys/select.h>
+#include <sys/socket.h>
+#include <arpa/inet.h>
+#include <wifi_manager/wifi_manager.h>
+#include <protocols/dhcpc.h>
+#include <protocols/dhcpd.h>
+#include <netutils/netlib.h>
+#include <tinyara/wifi/wifi_utils.h>
+
+/**
+ * Enum, definitions and variables
+ */
+typedef enum {
+	DHCP_ERR = -1,
+	DHCP_OK,
+	DHCP_EXIST,
+} dhcp_status_e;
+
+#define WIFIMGR_IP4_ZERO 0
+#define WIFIMGR_MAC_ZERO {0, 0, 0, 0, 0, 0}
+
+typedef void (*dhcp_sta_joined)(dhcp_evt_type_e type, void *data);
+
+/**
+ * Internal APIs
+ */
+void dhcps_inc_num(void);
+void dhcps_dec_num(void);
+void dhcps_reset_num(void);
+uint8_t dhcps_get_num(void);
+
+#ifndef CONFIG_WIFIMGR_DISABLE_DHCPS
+/* wifi manager stores only one mac address of a node while it is running on softap mode
+ * because most of wi-fi chipset that ported to TizenRT supports to join a device.
```

Comment:
I think "chipset" -> "chipsets", "that ported" -> "that are ported", and "supports to join" -> "support joining" are correct.
How do you think?

Reply:
Modified.
Though these sentences are ambiguous, changed them. Please check the new.
Thanks for the comment.